testing/sampletests/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/GoogleCloudPlatform/golang-samples/testing/sampletests
+
+go 1.12
+
+require (
+	github.com/google/go-cmp v0.5.2
+	github.com/jstemmer/go-junit-report v0.9.1
```

Comment:
I think there is a replace in the root go.mod we should copy over here.

Reply:
Whoops, good catch!